src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier c = new ColorSupplier();
```

Comment:
c - is bad naming, write full name

Reply:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/855c7fd5-fa62-418b-821b-2ce51c2973ef)

Reply:
not fixed